tools/stress-cluster/chaos/examples/network-stress-example/.gitignore

```diff
@@ -0,0 +1 @@
+generatedValues.yaml
```

Comment:
We have generally avoid multiple gitignore files can this be added to the root?

Reply:
It can but it's supposed to serve as an example that people can copy/paste for their stress tests. Do you think we should just add this file to all repository roots instead?

Reply:
Or perhaps find a common directory pattern and write files under that. Something like bin or obj are common directories that are in all our gitignore files.